src/App.jsx

```diff
@@ -14,45 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const def = [...goodsFromServer];
+  const [goods, setGoods] = useState(def);
+  const [sort, setSort] = useState('none');
+  const sortAlphabetically = () => {
+    setGoods([...goods].sort((a, b) => b.localeCompare(a)));
+    setSort('alphabetically');
+  };
+
+  const sortByLength = () => {
+    setGoods([...goods].sort((a, b) => a.length - b.length));
+    setSort('bylength');
+  };
+
+  const reverseGoods = () => {
+    setGoods([...goods].reverse());
+    setSort('reverse');
+  };
+
+  const resetGoods = () => {
+    setGoods(def);
+    setSort('none');
+  };
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={sort === 'alphabetically'
+            ? 'button is-info'
```

Comment:
create a variable like `SORT_ALPHABET = 'alphabet'` and use this kind of variables instead of string